crates/bevy_animation/src/lib.rs

```diff
@@ -123,23 +123,38 @@ impl AnimationClip {
     }
 }
 
+/// Repetition behavior of the animation.
+#[derive(Reflect, Copy, Clone)]
+pub enum RepeatAnimation {
+    /// Repeat forever.
+    Forever,
+    /// Repeat forever.
+    Once,
```

Comment:
Two things:
* RepeatAnimation::Once sounds to me like the animation plays twice (once then repeats once)
* This comment is wrong and doesn't help clarify the intended behavior :)

Reply:
>  RepeatAnimation::Once sounds to me like the animation plays twice (once then repeats once)

Agreed :)